recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,7 @@
+# need macOS 10.15 for aligned_alloc
+c_stdlib_version:  # [osx and x86_64]
+  - 10.15          # [osx and x86_64]
```

Comment:
@lpsinger, if you're patching out the need for macOS 10.15, please remove this change and regenerate the feedstock.